client/src/api/LowLevelApiClient.js

```diff
@@ -43,13 +43,15 @@ export default class LowLevelApiClient {
     };
     const out = fetch(endpoint, fetchConfig).catch(error => {
       return new Promise((resolve, reject) => {
-        reject({
-          response: {
-            status: 503,
-            statusText: error.name,
-            exception: error.message
-          }
-        });
+        reject(
```

Comment:
We need to test that API failures are still resolved correctly following this change. Move this commit into a separate PR so it can be tested more carefully.

Reply:
Will do